pkg/workers/konnectors/konnector.go

```diff
@@ -162,33 +173,57 @@ func Worker(ctx context.Context, m *jobs.Message) error {
 	scanOut := bufio.NewScanner(cmdOut)
 	scanOut.Buffer(nil, 256*1024)
 
-	go doScanOut(jobID, scanOut, domain)
+	var msgChan = make(chan konnectorMsg)
+	var messages []konnectorMsg
```

Comment:
I had the idea that in the future we will need to store progress & warning messages in konnectors.result.